builds.yml

```diff
@@ -138,6 +140,11 @@ features:
       - src: ./app/build-types/flask/images/
         dest: images
       - ./{app,shared,ui}/**/flask/**
+  keyring-snaps:
+    env:
+      - KEYRING_SNAPS_REGISTRY_URL: https://metamask.github.io/keyring-snaps-registry/prod/registry.json
```

Comment:
What's the long term plan for the registry? Have you been talking with @dayksx about fitting this into the permissionless contract?

Reply:
For now the decision was to use github for the registry.